packages/app-content-pages/src/screens/Publications/components/Publication/Publication.js

```diff
@@ -8,43 +8,56 @@ const StyledBox = styled(Box)`
   overflow: hidden;
 `
 
-const Placeholder = (
-  <Box
-    align='center'
-    justify='center'
-    background='brand'
-    height='100%'
-    width='100%'
-  >
-    <ZooniverseLogo size='50%' />
-  </Box>
-)
+const Placeholder = ({ id }) => {
+  return (
+    <Box
+      align='center'
+      justify='center'
+      background='brand'
+      height='100%'
+      width='100%'
+    >
+      <ZooniverseLogo size='50%' id={id} />
+    </Box>
+  )
+}
 
-function Publication (props) {
-  const { authors, avatarSrc, className, title, url, year } = props
+function Publication({
+  authors = '',
+  avatarSrc = '',
+  className = '',
+  title = '',
+  url = '',
+  year = ''
+}) {
   const displayString = [title, authors, year].filter(v => v).join(', ')
 
   return (
     <Box
       className={className}
+      data-testid='publication-test-element'
       direction='row'
       gap='small'
       margin={{ bottom: 'small' }}
     >
       <StyledBox height='50px' width='50px' flex={false}>
-        {!avatarSrc && Placeholder}
+        {!avatarSrc && <Placeholder id={displayString} />}
         {avatarSrc && (
           <Media
             height={50}
             src={avatarSrc}
-            placeholder={Placeholder}
+            placeholder={<Placeholder id={displayString} />}
```

Comment:
Possible invalid value for `id` here too. 

Browsers ignore invalid HTML. I’ll check what the rules are for identifiers.